app/(tabs)/(touren)/createManualStage.tsx

```diff
@@ -0,0 +1,200 @@
+          onChangeText={(nextValue) => setStageTitle(nextValue)}
+          maxLength={20}
+        />
+      </Layout>
+    ),
+    [stageTitle],
+  );
+
+  //switching between done and edit
+
+  const changingTitleButton = useMemo(
+    () => (
+      <Button
+        appearance="ghost"
+        onPress={() => setTitleBeingChanged((prevState) => !prevState)}
+      >
+        {titleBeingChanged ? (
+          <FontAwesomeIcon icon="check" size={25} />
+        ) : (
+          <FontAwesomeIcon icon="edit" size={25} />
+        )}
+      </Button>
+    ),
+    [titleBeingChanged],
+  );
+
+  // Update header input ot text based on if title or edit state changes
+  useEffect(() => {
+    navigation.setOptions({
+      headerTitle: () =>
+        titleBeingChanged ? (
+          titleInput
+        ) : (
+          <Text category="h4">{stageTitle}</Text>
+        ), // Switch between input and title
+      headerRight: () => changingTitleButton, // Set the button on the right
+      headerTitleAlign: "center", // Keep the title centered
+    });
+  }, [
+    navigation,
+    titleInput,
+    changingTitleButton,
+    titleBeingChanged,
+    stageTitle,
+  ]);
+
+  const startCoordInput = (
+    <CoordinateInput
+      lat={startLat}
+      lon={startLon}
+      setLat={setStartLat}
+      setLon={setStartLon}
+      date={startDate}
+      setDate={setStartDate}
+    />
+  );
+
+  const endCoordInput = (
+    <CoordinateInput
+      lat={endLat}
+      lon={endLon}
+      setLat={setEndLat}
+      setLon={setEndLon}
+      date={endDate}
+      setDate={setEndDate}
+    />
+  );
+
+  const onSubmitEtappe = async () => {
+    try {
+      await createManualStageFn(
+        stageTitle,
+        startLat + ", " + startLon,
+        endLat + ", " + endLon,
+        startDate,
+        endDate,
+        tourId,
+      );
+      router.back();
+    } catch (err) {
+      if (err instanceof Error) {
+        Alert.alert("Error", err.message);
+      } else {
+        Alert.alert("Unknown Error", "An unexpected error occurred.");
+      }
+    }
+  };
+
+  return (
+    <Layout style={styles.layout} level="2">
+      <ButtonGroup
+        style={{
+          borderRadius: 10,
+        }}
+      >
+        <Button style={styles.button} onPress={() => {}}>
+          <FontAwesomeIcon icon="compass" size={25} />
+        </Button>
+        <Button style={styles.button} onPress={() => {}}>
+          <FontAwesomeIcon icon="map-pin" size={25} />
+        </Button>
+        <Button style={styles.button} onPress={() => {}}>
+          <FontAwesomeIcon icon="city" size={25} />
+        </Button>
+      </ButtonGroup>
+
+      <Layout style={styles.cardsContainer} level="2">
+        <CardComponent title="Start" form={startCoordInput} />
+
+        <CardComponent title="Ende" form={endCoordInput} />
+      </Layout>
+      <ButtonGroup
+        style={{
+          borderRadius: 10, // Add a group-level border radius
+        }}
+      >
```

Comment:
```suggestion
      <ButtonGroup>
```

style existiert nicht in ButtonGroup